tests/osaka/eip7692_eof_v1/eip3540_eof_v1/test_extcode.py

```diff
@@ -23,15 +23,15 @@ def test_legacy_calls_eof_sstore(
 ):
     """Test EXTCODE* opcodes calling EOF and legacy contracts"""
     env = Environment()
-    address_eof_contract = pre.deploy_contract(
-        Container(
-            sections=[
-                Section.Code(
-                    code=Op.RJUMP[0] + Op.STOP,
-                )
-            ]
-        )
+    eof_code = Container(
```

Comment:
Can use `Container.Code(code)` here

Reply:
I never remember how it was... we need EEST/EOF cheatsheet.

Reply:
`Container.Code()` doesn't allow you to name the container.

Reply:
you do not name the container here.